pkg/check/postage/postage.go

```diff
@@ -0,0 +1,26 @@
+package postage
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/ethersphere/beekeeper/pkg/bee"
+)
+
+func GetDefaultBatchID(ctx context.Context, c *bee.Client, sleep time.Duration) (string, error) {
```

Comment:
i dont think this should be in the `check` folder, as the check folder is where we put actual correctness tests or other scenarios, and this falls under a helper method. having it in the api/postage might be a better place

Reply:
ok 👍